src/azure-cli/azure/cli/command_modules/cosmosdb/_params.py

```diff
@@ -75,3 +85,118 @@ def load_arguments(self, _):
+
+    with self.argument_context('cosmosdb mongodb collection') as c:
+        c.argument('account_name', options_list=['--account', '-a'], help="Cosmosdb account name", id_part=None)
+        c.argument('database_name', options_list=['--database', '-d'], help="Database name")
+        c.argument('collection_name', options_list=['--collection', '-n'], help="Collection name")
+        c.argument('shard_key_path', options_list=['--shard'], help="Sharding key path.")
+        c.argument('indexes', options_list=['--idx'], type=shell_safe_json_parse, completer=FilesCompleter(), help='Indexes, you can enter it as a string or as a file, e.g., --idx @indexes-file.json or ' + MONGODB_INDEXES_EXAMPLE)
+        c.argument('throughput', help='Offer Throughput (RU/s). Default value is 400')
+
+# Cassandra
+    with self.argument_context('cosmosdb cassandra keyspace') as c:
+        c.argument('account_name', options_list=['--account', '-a'], help="Cosmosdb account name", id_part=None)
+        c.argument('keyspace_name', options_list=['--keyspace', '-n'], help="Keyspace name")
+        c.argument('throughput', help='Offer Throughput (RU/s). Default value is 400')
+
+    with self.argument_context('cosmosdb cassandra table') as c:
+        c.argument('account_name', options_list=['--account', '-a'], help="Cosmosdb account name", id_part=None)
+        c.argument('keyspace_name', options_list=['--keyspace', '-k'], help="Keyspace name")
+        c.argument('table_name', options_list=['--table', '-n'], help="Table name")
+        c.argument('default_ttl', options_list=['--ttl'], type=int, help='Default TTL (https://docs.microsoft.com/en-us/azure/cosmos-db/time-to-live)')
+        c.argument('schema', type=shell_safe_json_parse, completer=FilesCompleter(), help='Schema, you can enter it as a string or as a file, e.g., --schema @schema-file.json or ' + CASSANDRA_SCHEMA_EXAMPLE)
+        c.argument('throughput', help='Offer Throughput (RU/s). Default value is 400')
+
+# Gremlin
+    with self.argument_context('cosmosdb gremlin database') as c:
+        c.argument('account_name', options_list=['--account', '-a'], help="Cosmosdb account name", id_part=None)
+        c.argument('database_name', options_list=['--database', '-n'], help="Database name")
+        c.argument('throughput', help='Offer Throughput (RU/s). Default value is 400')
+
+    with self.argument_context('cosmosdb gremlin graph') as c:
+        c.argument('account_name', options_list=['--account', '-a'], help="Cosmosdb account name", id_part=None)
+        c.argument('database_name', options_list=['--database', '-d'], help="Database name")
+        c.argument('graph_name', options_list=['--graph', '-n'], help="Graph name")
+        c.argument('partition_key_path', options_list=['--part'], help='Partition Key Path, e.g., \'/properties/name\'')
+        c.argument('default_ttl', options_list=['--ttl'], type=int, help='Default TTL (https://docs.microsoft.com/en-us/azure/cosmos-db/time-to-live)')
```

Comment:
Provide more detailed message for TTL. 
Please describe when --ttl is given, what will happen and when it is not provide, what will happen.

Reply:
link already attached

Reply:
But users have to click the link and come back to the command to decide what they will set for ttl argument. I think link is just a supplement for help message. It will be better to make users understand each command and argument just through help message.